Birthdays/Program.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Birthday
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            BirthdayManager("birthdays.txt");
+        }
+
+        public static Dictionary<string, Dictionary<string, int>> ReadBirthdaysFile(string path)
+        {
+            using(StreamReader file = new StreamReader(path)) 

```

Comment:
Возникает проблема с кодировкой. Исходный (тот, который был приложен к заданию) файл в кодировке WIn-1251. ОДнако вы делали задание для .NET Core, которая поддерживает немного кодировок для мультиплатформенности. В частности, WIn-1251 не поддерживает. То есть без изменения кодировки файла birthdays.txt не обойтись. Самое простое- перевести его в UTF-8 и  приязать к этой кодировке код.
```suggestion
            using (StreamReader file = new StreamReader(path, System.Text.Encoding.UTF8))

```